packages/language-server/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import { createSimpleProject } from '@volar/language-server/lib/project/simpleProject';
+import { createConnection, createServer } from '@volar/language-server/node';
+import { create as createCssService } from 'volar-service-css';
+
+// MEMO: Maybe @volar/language-server and volar-service-css are not needed. We may only need vscode-css-languageservice.
+
+const connection = createConnection();
+const server = createServer(connection);
+
+connection.listen();
+
+connection.onInitialize((params) => {
+  const cssService = createCssService();
+
+  // Disable rename provider due to conflict with ts-plugin
+  // TODO: Allow rename operations that do not conflict with ts-plugin
+  // TODO: Do not disable provider in CSS files that are not CSS Modules
+  delete cssService.capabilities.renameProvider;
+  // Disable references provider due to conflict with ts-plugin
+  // TODO: Allow references operations that do not conflict with ts-plugin
+  // TODO: Do not disable provider in CSS files that are not CSS Modules
+  delete cssService.capabilities.referencesProvider;
```

Comment:
Some language features are disabled here.

Reply:
If not disabled, the class selector rename will be broken. If a user tries to rename `a_1`, it will accidentally rename `.a_1`.


<img width="343" alt="image" src="https://github.com/user-attachments/assets/1a8e8a01-651c-462c-b91b-421f934c7c57" />

Also, "Find All References" will show both `.a_1` and `a_1`.

<img width="540" alt="image" src="https://github.com/user-attachments/assets/a150fd07-3f6a-4e64-9d40-ea0c386d0188" />